java/dev/checkstyle/checkstyle.xml

```diff
@@ -94,9 +81,6 @@
             <property name="tokens" value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
         </module>
         <module name="NeedBraces"/>
-        <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
```

Comment:
why this removal?

Reply:
This has been removed in 8.2: https://checkstyle.sourceforge.io/releasenotes.html#Release_8.2 Third point in the list.